src/deckardcain.js

```diff
@@ -20,7 +20,7 @@ const REFRACT_PARSE_RESULT_ELEMENT_YAML = /[\uFEFF]?\n?\s*element: ?["']parseRes
  * @returns {string|null} Media type of given file.
  */
 function identify(source) {
-  if (source.match(API_BLUEPRINT_HEADER)) {
+  if (source.substring(0, 11).match(API_BLUEPRINT_HEADER)) {
```

Comment:
Metadata section should start at the beginning of the document - https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#definition-3